sdk/formrecognizer/azure-ai-formrecognizer/MIGRATION_GUIDE.md

```diff
@@ -666,4 +666,4 @@ For additional samples please take a look at the [Form Recognizer Samples][sampl
 
 [readme]: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/formrecognizer/azure-ai-formrecognizer/README.md
 [samples_readme]: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/formrecognizer/azure-ai-formrecognizer/samples/README.md
-[fr_labeling_tool]: https://aka.ms/azsdk/formrecognizer/labelingtool
\ No newline at end of file
+[fr_labeling_tool]: https://aka.ms/azsdk/formrecognizer/formrecognizerstudio
```

Comment:
Do we need to update the sentence where we link to this to say "FR studio" instead of "labeling tool"?

Reply:
I did change it to say FR Studio

Reply:
I'm talking about L598

Reply:
Ah got it...I updated the wording. Let me know if that's better!